bridge/engine/EngineStreamDirector.h

```diff
@@ -697,7 +703,7 @@ class EngineStreamDirector
         const auto participantStreamsItr = _participantStreams.find(endpointIdHash);
         if (participantStreamsItr == _participantStreams.end())
         {
-            return lowQuality;
+            return dropQuality;
```

Comment:
wdyt about this one. It seems reasonable to me but perhaps there is something I missed on how Director works.